packages/neo-one-smart-contract-lib/src/ownership/withTransferableOwnership.ts

```diff
@@ -0,0 +1,77 @@
+// tslint:disable readonly-keyword readonly-array no-object-mutation strict-boolean-expressions no-unnecessary-field-initialization
+
+import { Address, constant, createEventNotifier, SmartContract } from '@neo-one/smart-contract';
+
+/**
+ * @title Transferable
+ * @dev A TransferableOwnership contract has an owner address, and provides basic authorization control
+ * functions, this simplifies the implementation of "user permissions".
+ */
+
+export function withTransferableOwnership<TBase extends Constructor<SmartContract>>(Base: TBase) {
+  abstract class TransferableOwnershipClass extends Base {
+    protected abstract mutableOwner: Address;
+    protected abstract mutableTransferTo: Address | undefined;
+
+    @constant
+    public get owner(): Address {
+      return this.mutableOwner;
+    }
+
+    private readonly notifyOwnershipTransferred = createEventNotifier<Address, Address>(
+      'contract ownership transferred',
+      'from',
+      'to',
+    );
+    private readonly notifyOwnershipTransferInitialized = createEventNotifier<Address, Address>(
+      'contract ownership transfer initiated',
+      'from',
+      'to',
+    );
+
+    private readonly notifyFailure = createEventNotifier('invalid call');
+
+    private readonly notifyOwnershipTransferCanceled = createEventNotifier('contract ownership transfer canceled');
+
+    public transferContract(from: Address, to: Address): boolean {
```

Comment:
Remove the `from` argument, it's unnecessary - `from` is always the current owner.

Reply:
👍